lib/Runtime/Debug/BreakpointProbe.cpp

```diff
@@ -12,6 +12,7 @@ namespace Js
         characterOffset(statement.statement.begin),
         byteOffset(statement.bytecodeSpan.begin)
     {
+        this->breakPointId = functionBody->GetScriptContext()->GetThreadContext()->GetDebugManager()->GetNextBreakpointId();
```

Comment:
Ok, changed

---
In reply to: [62776574](https://github.com/Microsoft/ChakraCore/pull/926#discussion_r62776574) [](ancestors = 62776574,62775405,62774317)